### ISSUE TITLE
Error: Argument for @NotNull parameter 'file' of com/intellij/openapi/vfs/VfsUtilCore.getRelativePath must not be null

### ISSUE THREAD

**Issue by memdev** [Haehnchen/idea-php-shopware-plugin#110]
Hi,

I am getting the following error when using the Shopware plugin (version 4.1.1) in PhpStorm 2019.2.5

Any ideas?

```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'file' of com/intellij/openapi/vfs/VfsUtilCore.getRelativePath must not be null
	at com.intellij.openapi.vfs.VfsUtilCore.$$$reportNull$$$0(VfsUtilCore.java)
	at com.intellij.openapi.vfs.VfsUtilCore.getRelativePath(VfsUtilCore.java)
	at de.espend.idea.shopware.util.SnippetUtil.getFileNamespaceViaPath(SnippetUtil.java:261)
	at de.espend.idea.shopware.util.TemplateUtil.getTemplateNameViaPath(TemplateUtil.java:271)
	at de.espend.idea.shopware.util.TemplateUtil.getTemplateName(TemplateUtil.java:280)
	at de.espend.idea.shopware.util.TemplateUtil.getTemplateName(TemplateUtil.java:262)
	at de.espend.idea.shopware.navigation.SmartyTemplateLineMarkerProvider.getImplementedBlocks(SmartyTemplateLineMarkerProvider.java:180)
	at de.espend.idea.shopware.navigation.SmartyTemplateLineMarkerProvider.collectSlowLineMarkers(SmartyTemplateLineMarkerProvider.java:73)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.queryProviders(LineMarkersPass.java:194)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.lambda$doCollectInformation$3(LineMarkersPass.java:96)
	at com.intellij.codeInsight.daemon.impl.Divider.divideInsideAndOutsideInOneRoot(Divider.java:79)
	at com.intellij.codeInsight.daemon.impl.LineMarkersPass.doCollectInformation(LineMarkersPass.java:91)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:55)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:429)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1106)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:422)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:591)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:537)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:59)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:421)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:397)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:204)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:395)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:161)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:177)
```

**Comment by tinect** [comment]
Same here

**Comment by tinect** [comment]
@Haehnchen Have you already taken care of it?

**Comment by Haehnchen** [comment]
fixed via #111 release will follow

**Comment by tinect** [comment]
This error still appears in 4.1.2
